cfme/v2v/infrastructure_mapping.py

```diff
@@ -0,0 +1,298 @@
+import attr
+
+from navmazing import NavigateToAttribute, NavigateToSibling
+from widgetastic.widget import View, ParametrizedView
+from widgetastic_manageiq import (
+    InfraMappingList,
+    InfraMappingTreeView,
+    MultiSelectList,
+    V2VPaginatorPane,
+)
+from widgetastic_patternfly import Button, SelectorDropdown, Text, TextInput
+from widgetastic.utils import WaitFillViewStrategy, ParametrizedLocator
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.exceptions import ItemNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from cfme.utils.update import Updateable
+from cfme.utils.version import Version, VersionPicker
+
+
+class InfrastructureMappingView(BaseLoggedInPage):
+    title = Text("#explorer_title_text")
+
+    @property
+    def in_explorer(self):
+        return self.logged_in_as_current_user and (
+            self.navigation.currently_selected
+            == ["Compute", "Migration", "Infrastructure Mappings"]
+        )
+
+    @property
+    def is_displayed(self):
+        return self.in_explorer
+
+
+class InfraMappingCommonButtons(InfrastructureMappingView):
+    back_btn = Button("Back")
+    next_btn = Button("Next")
+    cancel_btn = Button("Cancel")
+    save = Button("Save")
+    create = Button("Create")
+    add_mapping = Button("Add Mapping")
+    remove_mapping = Button("Remove Selected")
+    remove_all_mappings = Button("Remove All")
+    mappings_tree = InfraMappingTreeView(tree_class="treeview")
+
+
+class ComponentView(View):  # noqa
```

Comment:
Q: why `#noqa` ?

Reply:
Oh I changed the name but forgot to remove #noqa